bikeshed/ReferenceManager.py

```diff
@@ -273,21 +273,28 @@ def getRef(self, linkType, text, spec=None, status=None, linkFor=None, error=Tru
             candidateFor, _, forPrefix = linkFor.partition("/") if "/" in linkFor else (None, None, '')
             forPrefix = forPrefix[:-1]
             candidates, _ = self.queryRefs(linkType="functionish", status="local", linkFor=candidateFor)
-            localUrls = list(set(c.url for c in candidates if c.text.startswith(forPrefix)))
-            if len(localUrls) == 1:
-                return localUrls[0]
+            localRefs = list(set(c for c in candidates if c.text.startswith(forPrefix)))
+            if len(localRefs) == 1:
+                return localRefs[0]
             # And repeat for non-locals
             candidates, _ = self.queryRefs(linkType="functionish", spec=spec, status=status, linkFor=candidateFor, export=export, ignoreObsoletes=True)
-            remoteUrls = list(set(c.url for c in candidates if c.text.startwith(forPrefix)))
-            if len(remoteUrls) == 1:
-                return remoteUrls[0]
-            if zeroRefsError and (len(localUrls) or len(remoteUrls)):
+            remoteUrls = list(set(c for c in candidates if c.text.startwith(forPrefix)))
+            if len(remoteRefs) == 1:
+                return remoteRefs[0]
+            if zeroRefsError and (len(localRefs) or len(remoteRefs)):
                 die("Too many possible method targets to disambiguate '{0}/{1}'. Please specify the names of the required args, like 'foo(bar, baz)', in the 'for' attribute.", linkFor, text)
 
         if failure == "text" or failure == "type":
             if spec and spec in self.anchorMacros:
                 # If there's a macro registered for this spec, use it to generate a ref.
-                return anchorMacros[spec].url + config.simplifyText(text)
+                return {
```

Comment:
Actually I think this can be removed?  I don't believe I support anchor macros right now.  It was a planned feature earlier, when `<pre class=anchors>` had a much more annoying and verbose JSON structure.